src/commands/android/subcommands/install/avd.ts

```diff
@@ -0,0 +1,134 @@
+  try {
+    const avdmanagerLocation = getBinaryLocation(sdkRoot, platform, 'avdmanager', true);
+    if (!avdmanagerLocation) {
+      showMissingBinaryHelp('avdmanager');
+
+      return false;
+    }
+
+    const sdkmanagerLocation = getBinaryLocation(sdkRoot, platform, 'sdkmanager', true);
+    if (!sdkmanagerLocation) {
+      showMissingBinaryHelp('sdkmanager');
+
+      return false;
+    }
+
+    const avdNameAnswer = await inquirer.prompt({
+      type: 'input',
+      name: 'avdName',
+      message: 'Enter a name for the AVD:'
+    });
+    const avdName = avdNameAnswer.avdName;
+
+    const installedSystemImages = await getInstalledSystemImages(sdkmanagerLocation, platform);
+    if (!installedSystemImages.result) {
+      return false;
+    } else if (!installedSystemImages.systemImages.length) {
+      Logger.log(colors.red('No system image is installed!'));
+      Logger.log(`Run ${colors.cyan('npx @nightwatch/mobile-helper android install --system-image')} to install system images.`);
+
+      return false;
+    }
+
+    const systemImageAnswer = await inquirer.prompt({
+      type: 'list',
+      name: 'systemImage',
+      message: 'Select the system image to use for AVD:',
+      choices: installedSystemImages.systemImages
+    });
+    const systemImage = systemImageAnswer.systemImage;
+
+    const deviceTypes: DeviceType[] = ['Nexus', 'Pixel', 'Wear OS', 'Android TV', 'Desktop', 'Others'];
+    const deviceTypeAnswer = await inquirer.prompt({
+      type: 'list',
+      name: 'deviceType',
+      message: 'Select the device type for AVD:',
+      choices: deviceTypes
+    });
+    const deviceType = deviceTypeAnswer.deviceType;
+
+    let cmd = `list devices -c | grep ${deviceTypesToGrepCommand[deviceType as DeviceType]}`;
+    const availableDevices = execBinarySync(avdmanagerLocation, 'avdmanager', platform, cmd);
+
+    if (!availableDevices) {
+      Logger.log(`${colors.red('No devices found!')} Please try again.`);
+
+      return false;
+    }
+    const availableDevicesList = availableDevices.split('\n').filter(device => device !== '');
+    const deviceAnswer = await inquirer.prompt({
+      type: 'list',
+      name: 'device',
+      message: 'Select the device profile for AVD:',
+      choices: availableDevicesList
+    });
+    const device = deviceAnswer.device;
+
+    Logger.log();
+    Logger.log('Creating AVD...');
+
+    cmd = `create avd -n '${avdName}' -k '${systemImage}' -d '${device}'`;
+    try {
+      const output = await execBinaryAsync(avdmanagerLocation, 'avdmanager', platform, cmd);
+      if (output?.includes('100% Fetch remote repository')) {
+        Logger.log();
+        Logger.log(`${colors.green('AVD created successfully!')}`);
+
+        return true;
+      }
+    } catch (err) {
+      if (typeof err === 'string') {
+        if (err.includes('already exists')) {
+          // AVD with the same name already exists. Ask user if they want to overwrite it.
```

Comment:
This catch block only handles the errors that are of type 'string' and contains 'already exists'. All other errors will never be logged since there is no logic for "else" anywhere.

The corresponding `catch` block will only handle the error if the `create avd` command with `--force` also fails.